doc/manual/source/protocols/store-path.md

```diff
@@ -20,9 +20,12 @@ where
 
 - `store-dir` = the [store directory](@docroot@/store/store-path.md#store-directory)
 
-- `digest` = base-32 representation of the first 160 bits of a [SHA-256] hash of `fingerprint`
+- `digest` = base-32 representation of the compressed to 160 bits [SHA-256] hash of `fingerprint`
 
-  This the hash part of the store name
+For the definition of the hash compression algorithm, please refer to the section 5.1 of
+the [Nix thesis](https://edolstra.github.io/pubs/phd-thesis.pdf), which also defines the
+specifics of base-32 encoding. Note that base-32 encoding processes the hash bytestring from
+the end, while base-16 processes in from the beginning.
```

Comment:
This is fine temporarily, but I rather move the text over so the manual is self-contained

Reply:
Maybe some other time. For now, I don't strive for the best documentation ever, I just want to make sure that this pages contains information and references necessary for independent implementation.

It took me quite some time to realize that base-32 in this document processes string backwards; I want to save this effort for other people.